syntax = "proto3";

package narupa.protocol.trajectory;

import "narupa/protocol/array.proto";
import "google/protobuf/struct.proto";

message FrameData {
  map<string, google.protobuf.Value> values = 1;
  map<string, narupa.protocol.ValueArray> arrays = 2;
}
